File: translations/codenet/deepseek_commented/Go2C/granite/Code/s336868078.c


#include <stdio.h>

int solve(int a, int b, int c) {
    int temp_;
    if (c == a + b) {
        temp_ = b + c;
    } else if (c - 1 > a + b) {
        int temp = (c - 1) - (a + b);
        temp_ = b + c - temp;
    } else {
        temp_ = b + c;
    }
    return temp_;
}

int main() {
    int a, b, c;
    scanf("%d %d %d", &a, &b, &c);

    int ans = solve(a, b, c);
    printf("%d\n", ans);
    return 0;
}

File: translations/codenet/uncommented/Go2P/gpt-4o/Code/s336868078.py
def solve(a, b, c):
    if c == a + b:
        temp_ = b + c
    elif c - 1 > a + b:
        temp = (c - 1) - (a + b)
        temp_ = b + c - temp
    else:
        temp_ = b + c

    return temp_

def main():
    a, b, c = map(int, input().split())
    ans = solve(a, b, c)
    print(ans)

if __name__ == "__main__":
    main()

# <END-OF-CODE>
